front/src/modules/pipeline-processes/components/Board.tsx

```diff
@@ -70,11 +71,14 @@ export function Board({
 }: BoardProps) {
   const [board, setBoard] = useRecoilState(boardColumnsState);
   const [items, setItems] = useRecoilState(boardItemsState);
+  const [selectedItemKeys, setSelectedItemKeys] = useRecoilState(
```

Comment:
We use a convention where the name of the atom has to match the name of the value and the setter, so here it would be : 
[selectedBoardItems, setSelectedBoardItems]